src/routes/transactions/mappers/multisig-transactions/multisig-transaction-execution-details.mapper.spec.ts

```diff
@@ -246,4 +249,32 @@ describe('MultisigTransactionExecutionDetails mapper (Unit)', () => {
       }),
     );
   });
+  it('should return a MultisigExecutionDetails object with no proposedByDelegate if not present', async () => {
```

Comment:
Super nitpicky: missing newline just before this test case 🙂

Reply:
Added in f18756f9d9f79e2f9041fb7434f64b8bcd7cdf6b.